skype-sender-exercise/skype-sender-post-message.py

```diff
@@ -0,0 +1,22 @@
+"Post message to Skype TestBot through API Endpoint"
+import requests
+import json
+from conf import channel
+from conf import API_KEY
```

Comment:
conf can be import in single line "import conf" and can able to set values as conf.{variable}